packages/lit-element/src/polyfill-support.ts

```diff
@@ -42,7 +42,12 @@ interface PatchableLitElement extends HTMLElement {
   renderOptions: RenderOptions;
 }
 
-globalThis.litElementPlatformSupport ??= ({
+// Note, explicitly use `var` here so that this can be re-defined when
+// bundled.
+// eslint-disable-next-line no-var
+var DEV_MODE = true;
```

Comment:
Seems bad that polyfill supports are being bundled together without iffe's such that this would error as a `const`, but afaiu that's safe because anytime we'd actually have conflicts between them they'd error, so probably not worth the time to fix now.